packages/ui/components/apps/AppCard.tsx

```diff
@@ -37,9 +37,11 @@ export function AppCard({ app, credentials, searchText, userAdminTeams }: AppCar
 
   const allowedMultipleInstalls = app.categories && app.categories.indexOf("calendar") > -1;
   const appAdded = (credentials && credentials.length) || 0;
-  const appInstalled = userAdminTeams?.length
-    ? userAdminTeams.length && appAdded >= userAdminTeams.length
-    : appAdded > 0;
+  const enabledOnTeams = !app.categories.some(
+    (category) => category === "calendar" || category === "conferencing"
+  );
```

Comment:
@zomars, I need to add a check for concurrent meetings when enabling team conferencing apps. This is just a part of the stack of PRs that would be nice to have in prod.